.gitignore

```diff
@@ -18,3 +18,6 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+.vscode
+*/vscode
```

Comment:
What is `*/vscode` ignoring? I've not seen VS Code put anything outside `.vscode`.